### PR TITLE
Integration/bs 316 bs 306

### PR DESCRIPTION
**Pull request by AbstractFruitFactory**

### CODE REVIEW

**Comment by MuncleUscles** [comment]
Tests appear to be failing

File: src/modules/universe/RadixNodeDiscoveryHardcoded.ts [hunk 1]
```diff
@@ -12,16 +12,22 @@ export default class RadixNodeDiscoveryHardcoded implements RadixNodeDiscovery {
...
+    private readonly bootstrapNodes: string[]
     
     /**
      * Creates an instance of radix node discovery from seed.
-     * @param bootstrapNode Full address to the rpc endpoint of a Radix node in the universe
+     * @param bootstrapNodes List of full addresses to the rpc endpoints of Radix nodes in the universe
      * @param ssl Whether to use SSL encryption for communication wiht the nodes
      */
     constructor(
-        readonly bootstrapNodes: string[],
```
Comment (MuncleUscles) [inline src/modules/universe/RadixNodeDiscoveryHardcoded.ts:22]: What's the reasoning behind this change?

File: src/modules/universe/RadixUniverse.ts [hunk 2]
```diff
@@ -119,6 +122,25 @@ export default class RadixUniverse {
         radixTokenManager.initialize(this.nativeToken)
     }
 
+    /**
+     * Bootstraps the universe using the universe config from a specified node.
+     */
```
Comment (MuncleUscles) [inline src/modules/universe/RadixUniverse.ts:127]: This is not accurate anymore

File: src/modules/universe/RadixNodeConnection.ts [hunk 3]
```diff
@@ -306,6 +306,14 @@ The atom may or may not have been accepted by the node.
             })
     }
 
+    public async getUniverse() {
```
Comment (MuncleUscles) [inline src/modules/universe/RadixNodeConnection.ts]: Any new public methods should have a tsdoc comment(also appreciated if you add it to any old methods that don't have it if you have a chance)

**Review by MuncleUscles — COMMENTED** [review]
Just a few notes about documentation

File: src/modules/universe/RadixNodeDiscoveryHardcoded.ts (hunk 1 above)
Comment (AbstractFruitFactory) [reply src/modules/universe/RadixNodeDiscoveryHardcoded.ts:22]: I reverted this

File: src/modules/universe/RadixNodeConnection.ts (hunk 3 above)
Comment (AbstractFruitFactory) [reply src/modules/universe/RadixNodeConnection.ts]: Noted. Although this method was not used anymore, so I removed it.

File: src/modules/universe/RadixUniverse.ts (hunk 2 above)
Comment (AbstractFruitFactory) [reply src/modules/universe/RadixUniverse.ts:127]: Changed.

**Review by MuncleUscles — APPROVED** [review]
LGTM

**Pull request merged**